pilot/pkg/model/push_context.go

```diff
@@ -773,7 +782,9 @@ func (ps *PushContext) updateContext(
 	var servicesChanged, virtualServicesChanged, destinationRulesChanged, gatewayChanged,
 		authnChanged, authzChanged, envoyFiltersChanged, sidecarsChanged bool
 
+	log.Infof(" @@@@ updateContext %v", pushReq)
```

Comment:
remove `@@@@`, or more likely just remove the entire message?

Reply:
Removed